src/Simulation.C

```diff
@@ -56,8 +56,8 @@ Simulation::Simulation(const YAML::Node& root_node)
 #endif
 
 #ifdef KOKKOS_ENABLE_HIP
-  // hipDeviceGetLimit(&default_stack_size, cudaLimitStackSize);
-  // hipDeviceSetLimit(cudaLimitStackSize, nalu_stack_size);
+  hipDeviceGetLimit(&default_stack_size, hipLimitMallocHeapSize);
+  hipDeviceSetLimit(hipLimitMallocHeapSize, nalu_stack_size);
```

Comment:
Excellent, I had thought that we believed these functions were not yet available. Are they newly available in 5.2 or later?

Reply:
We need 5.3 or later for these

Reply:
Given that 5.4 seems to be the default in spack-manager, I think we're ok

Reply:
We can put a hard constraint on nalu-wind in spack that makes it so it won't build with anything less than 5.3 when needed.